extensions/wikia/AdEngine/js/slot/inContent.js

```diff
@@ -0,0 +1,58 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.inContent', [
+	'ext.wikia.adEngine.adTracker',
+	'wikia.log',
+	'wikia.window'
+], function (adTracker, log, win) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.slot.inContent',
+		selector = '#mw-content-text > h2',
+		header;
+
+	/**
+	 * Adds dynamically new slot in the right place and sends tracking data
+	 */
+	function init(slotName) {
+		var adHtml = '<div id="' + slotName + '" class="wikia-ad default-height"></div>',
+			logMessage,
+			logWikiData = '(wikiId: ' + win.wgCityId + ' articleId: ' + win.wgArticleId + ')',
+			$header,
+			slotNameGA;
+
+		if (!slotName) {
+			log('slotName is falsy', 'error', logGroup);
+			return;
+		}
+
+		slotNameGA = slotName.toLowerCase();
+
```

Comment:
We already have tracking for that:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/AdEngine2.js#L117
So it would be tracked here:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/SlotTracker.js#L134

**BUT** by default it'll go to `ad/register/DirectGpt/other`. I'd add a new group (`incontent`?) here:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/SlotTracker.js#L16
(in order to make `ad/register/DirectGpt/incontent`)